src/routes/media.ts

```diff
@@ -23,4 +23,8 @@ router.get('/imdbid', async(ctx) => {
   await MediaController.getByImdbID(ctx);
 });
 
+router.get('/getall', async(ctx) => {
```

Comment:
Open to changing the name of this, couldn't think of anything better

Reply:
`all` might be misleading since it still isn't used for series info. What about `/video` since it is the route for all video results?